### PR TITLE
Use eslint formatter instead of standard typescript formatter

### PR DESCRIPTION
**Pull request by janpaul123**
It can fix a bit more stuff!

### CODE REVIEW

File: workspace.code-workspace [hunk 1]
```diff
@@ -20,11 +20,13 @@
...
 	"extensions": {
 		"recommendations": [
 			"matklad.rust-analyzer",
-			"vadimcn.vscode-lldb"
+			"vadimcn.vscode-lldb",
+			"dbaeumer.vscode-eslint"
 		]
 	},
-}
+}
```
Comment (disambiguator) [inline workspace.code-workspace:32]: I recommend setting your VSCode to auto add trailing newlines to prevent these kinds of diffs.

**Review by disambiguator — COMMENTED** [review]
Do you have an example of a way this lints better?

File: workspace.code-workspace (hunk 1 above)
Comment (janpaul123) [reply workspace.code-workspace:32]: Ah dang; tnx for the reminder!!

Comment (janpaul123) [reply workspace.code-workspace]: Added a commit to fix this in the workspace! 😄

**Comment by pankdm** [comment]
Do you also want to run the whole codebase through this formatter to reduce future noise from PRs?

**Comment by janpaul123** [comment]
> Do you have an example of a way this lints better?

Yeah! As far as I can tell this is equivalent to running `yarn lint --fix` manually. For example, this fixed [these](https://github.com/Zaplib/zaplib/runs/5150592282?check_suite_focus=true) linter errors, while the default formatter did not.

> Do you also want to run the whole codebase through this formatter to reduce noise from future PRs?

This doesn't change our linting, it just makes it easier to run from VSCode.

**Review by disambiguator — APPROVED** [review]

**Pull request merged**